xmake/modules/package/manager/find_package.lua

```diff
@@ -176,5 +168,5 @@ function main(name, opt)
     end
 
     -- ok?
-    return result
+    return result, manager_name, package_name
```

Comment:
We should return the package manager that was actually detected, because the passed name parameter may not contain `xxx::`.

https://github.com/xmake-io/xmake/blob/d79274d2e2d57a873e69463848f0ba4e1074dd7c/xmake/modules/package/manager/find_package.lua#L67-L76

Reply:
Good point, thanks. Getting it from there.